src/main/java/com/autotune/analyzer/Analyzer.java

```diff
@@ -52,7 +52,7 @@ public static void addServlets(ServletContextHandler context) {
         context.addServlet(ListStacks.class, ServerContext.LIST_STACKS);
         context.addServlet(ListStackLayers.class, ServerContext.LIST_STACK_LAYERS);
         context.addServlet(ListStackTunables.class, ServerContext.LIST_STACK_TUNABLES);
-        context.addServlet(ListAutotuneTunables.class, ServerContext.LIST_AUTOTUNE_TUNABLES);
+        context.addServlet(ListKruizeTunables.class, ServerContext.LIST_AUTOTUNE_TUNABLES);
```

Comment:
@dinogun - I think this change needs to be reflected in some tests, API design doc and kruize-demos (autotune minikube script). We can make these changes in separate PRs. Approving this PR, Thank you.

Reply:
tests and API docs now updates. Will raise a PR for kruize-demos